10.0/Dockerfile-onbuild

```diff
@@ -0,0 +1,15 @@
+FROM camptocamp/odoo-project:10.0-latest
+MAINTAINER Camptocamp
+
+# intermediate images should help speed up builds when only local-src, or only
+# external-src changes
+ONBUILD COPY ./src /opt/odoo/src
+ONBUILD COPY ./external-src /opt/odoo/external-src
+ONBUILD COPY ./local-src /opt/odoo/local-src
+ONBUILD COPY ./data /opt/odoo/data
+ONBUILD COPY ./songs /opt/odoo/songs
+ONBUILD COPY ./setup.py /opt/odoo/
+ONBUILD COPY ./VERSION /opt/odoo/
+ONBUILD COPY ./migration.yml /opt/odoo/
+# need to be called at the end, because it installs . and src
+ONBUILD RUN cd /opt/odoo && pip install -r src_requirements.txt
```

Comment:
Curious about this one though, does it mean that the libs will be installed only when you start the container?

Reply:
For now yes, but as most of lib are already here, so it's very fast as it just run a pip install for installing odoo in dev mode. So it work but not very clean solution

After inspecting the docker container the pip install add this 3 files in the image:
/usr/local/lib/python2.7/dist-packages/easy-install.pth
/usr/local/lib/python2.7/dist-packages/odoo.egg-link
/usr/local/bin/odoo

and for sure also add the odoo.egg-info in src folder (but this is mounted)

After some test I see to possibility to "pre-installing odoo".
- adding this 3 files
- downloading odoo (in a RUN cmd) installing it in dev mode and removing it.

I can do it here or it a specific akretion image based on "-core" image. What do you think?